Dockerfile

```diff
@@ -6,15 +6,15 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-FROM alpine:3.11.3
+FROM registry.access.redhat.com/ubi8:8.1-397
```

Comment:
Recommend adding a comment line above this:

`# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8`

That way when it's time to update this image, you can just run https://github.com/redhat-developer/codeready-workspaces/blob/master/product/updateBaseImages.sh and it'll automatically find the latest AND push a commit for you too.

Reply:
If you wanted to do more work here you could use ubi8-minimal (smaller base image) and then use `microdnf` to install the stuff you actually need in the container, like `microdnf -y install curl openssl`